core/geometry/src/polyface/AuxData.ts

```diff
@@ -270,10 +270,11 @@ export class PolyfaceAuxData {
           case AuxChannelDataType.Normal: {
             inverseRot = inverseRot ?? rot.inverse();
```

Comment:
Instead of having `inverseRot` be a `let` variable at the top of this function, declare it as a `const` right here. If you do that, the check for `inverseRot` being defined inside `transformPoints` should no longer be necessary. You also won't have to provide a type, since it is getting its value when it is defined.